### ISSUE TITLE
Rooms and private chats

### ISSUE THREAD

**Issue by IzzySoft** [Nheko-Reborn/nheko#1718]
Whenever I start a new private chat, Nheko is creating a room which I then have to convert to a private chat (`/converttodm`), But even when doing that, the room is left behind in the "rooms" tab. Am I doing something wrong there? How can I 1) create a direct-chat directly and 2) get rid of the rooms left when having them converted to DM? Thanks ahead for advice!

**Comment by IzzySoft** [comment]
Thanks @deepbluev7! 2 questions here:

1. is it safe to delete the "orphaned" room after conversion to DM? Currently, Nheko still shows those.
2. will the next release automatically clean up those left-over rooms – and is there an ETA for a new release?

**Comment by deepbluev7** [comment]
If you mark a room as dm, it shouldn't leave an orphan room, it just sets a flag for the current room. And no, the next release will not automatically clean them up, sorry...

**Comment by IzzySoft** [comment]
Thanks! I can live with the latter if I know it's safe to clean up the former and how to do that.

Those "orphaned rooms" are only shown in Nheko, they are not visible when e.g. logging in via Element Web. But within Nheko, those "rooms" are kept in-sync with the DMs as if they were pointers/symlinks, i.e. the DMs are "duplicated" as rooms (meaning, they exist in both tabs with identical content). Even as far as that, if I select one of those "rooms" and then switch to the "DM" tab, the corresponding chat is selected there and vice versa.

TL;DR: each of my "DM contacts" in Nheko still shows up as room in addition. They don't show up as room in Element. So in Nheko, the "rooms" tab looks exactly like the "direct chats" tab with just the "real rooms" added.

I've tried to log out and log in again in the hope this would clean it up, but it did not. Is this a separate issue to open? Like "Do not show direct chats in the group tab" (or have an option for that)?

**Comment by redsky17** [comment]
DMs will still show up in the room list as a normal room would.  The little people icon is a filter.  It sounds like what you’re describing isn’t so much an orphan as it is the same room showing up in the global (unfiltered) room list.  Unless you see two entries of this room in the unfiltered room list?

**Comment by IzzySoft** [comment]
> DMs will still show up in the room list as a normal room would. The little people icon is a filter.

That matches my observation, thanks! So nothing I need to "clean up" (in the sense of "deleting remains"). In that case please let me rephrase: is there a way to keep DMs out of the rooms list? Else the latter can become a bit messy with many DMs and few rooms. I could not find anything for that in the settings (but noticed that even DMs have "room settings" like "anyone can join", which makes no sense to me – but that might be due to my limited Matrix experience). An "all" tab is fine, but a dedicated room tab (like Element seems to have it) should be there IMHO.

> Unless you see two entries of this room in the unfiltered room list?

Only for one contact. Same address for both, but different room contents. No idea what that is, but if I can hide DM channels from the rooms' tab I can easily ignore that as one of them will "age out".
